runbooks/how-to-rerun-failed-pipeline-load.md

```diff
@@ -0,0 +1,65 @@
+# How to Re-run a Failed BFD Pipeline Load
+
+Follow this runbook to successfully reload data via BFD pipeline after a failed run.
+
+Note: If there are pending deployments or db migrations, make sure those finish before running these steps.
+
+1. SSH into the AWS ETL EC2 instance for a given environment ```bfd-<test/prod/prod-sbx>-etl``` with ```ssh -i <local ssh key> <aws username>@<EC2 IP Address>```. The entry point is the specific AWS user home directory.
+
+2. Confirm the pipeline has failed to load data. 
+
+    - In AWS S3, the RIF folder (i.e. ```<yyyy>-<MM>-<dd>T<HH>:<mm>:<ss>Z```) containing the data for reloading will still be in 'Incoming' with the file S3 file structure as:
```

Comment:
> - In AWS S3 [...]

This is just fine for me and likely others on the team. With a contributor less familiar with the process, it might be helpful to point out that the specific ETL pipeline's S3 bucket is configured in the same configuration file along with the values for idempotency mode etc under `S3_BUCKET_NAME`. This could be found with a one-liner, e.g.

```
grep S3_BUCKET_NAME /bluebutton-data-pipeline/bfd-pipeline-service.sh | cut -f2 -d=
```

Reply:
Updated to include this.